gnomad_qc/v3/create_release/get_het_nonref_variants_to_fix.py

```diff
@@ -0,0 +1,181 @@
+# NOTE:
+# This script is needed for the v3.1.2 release to correct a a problem discovered in the gnomAD v3.1 release.
+# The fix we implemented to correct a technical artifact creating the depletion of homozygous alternate genotypes
+# was not performed correctly in situations where samples that were individually heterozygous non-reference had both
+# a heterozygous and homozygous alternate call at the same locus.
+
+import argparse
+import logging
+
+import hail as hl
+
+from gnomad.resources.resource_utils import DataException
+from gnomad.utils.file_utils import file_exists
+from gnomad.utils.slack import slack_notifications
+from gnomad.utils.sparse_mt import densify_sites
+from gnomad.utils.vcf import SPARSE_ENTRIES
+
+from gnomad_qc.v3.resources.basics import get_gnomad_v3_mt
+from gnomad_qc.v3.resources.release import release_sites
+from gnomad_qc.v3.resources.annotations import get_info, last_END_position
+from gnomad_qc.slack_creds import slack_token
+
+
+logging.basicConfig(
+    format="%(asctime)s (%(name)s %(lineno)s): %(message)s",
+    datefmt="%m/%d/%Y %I:%M:%S %p",
+)
+logger = logging.getLogger("get_impacted_variants")
+logger.setLevel(logging.INFO)
+
+
+def get_het_non_ref_impacted_var(
+    mt: hl.MatrixTable, info_ht: hl.Table, freq_ht: hl.Table
+) -> None:
+    """
+    Filter to variants where homalt hotfix incorrectly adjusts het nonref genotype calls.
+
+    :param hl.MatrixTable mt: Raw, split MatrixTable annotated with original genotype het nonref status.
+    :param hl.Table info_ht: Info Hail Table containing AS_lowqual information.
```

Comment:
this isn't getting used in this function, is it? we should probably remove AS_lowqual variants after densifying anyway, right?

Reply:
Nope, no longer used. AS_lowqual are already removed from the freq_ht, so I don't think I need to remove them anywhere

Reply:
So I don't even need the info HT actually